.github/workflows/build.yml

```diff
@@ -0,0 +1,170 @@
+name: Build FreeOrion and run unit tests
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+  push:
+    branches:
+      - master
+jobs:
+  build-linux:
+    name: Build FreeOrion on Ubuntu 18.04 (Bionic)
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        id: build-linux-install-cache-id
+        with:
+          path: ${{ runner.temp }}/cache-build-linux-install
+          key: build-linux-install-cache-key-v1.0
+      - uses: airvzxf/cache-anything-new-action@v1.0
+        with:
+          script: 'install-build.sh'
+          is_cached: ${{ steps.build-linux-install-cache-id.outputs.cache-hit }}
+          cache: ${{ runner.temp }}/cache-build-linux-install
+          snapshot: '/'
+          exclude: '/boot /data /dev /mnt /proc /run /sys'
+      - name: Prepare ccache timestamp
+        id: build-linux-ccache-timestamp
+        shell: cmake -P {0}
+        run: |
+          string(TIMESTAMP current_date "%Y-%m-%d-%H;%M;%S" UTC)
+          message("::set-output name=timestamp::${current_date}")
+      - name: ccache files
```

Comment:
I'd recommend rewording it to clarify that `ccached` is not a typo but a tool name.

`Cache files with ccache`

Reply:
OK, fixed.